### ISSUE TITLE
TargetDisk

### ISSUE THREAD

**Issue by Audigy35** [PowerShellCrack/PSDWizardNew_CustomPages#1]
i added the targetdisk pane to the  psd wizard followed all the instructions but when the wizard loads it doenst detect any disk in the disk viewer or
volume viewer like
like in this image:

![image](https://github.com/user-attachments/assets/c4bbcb35-7462-4f35-bc85-969e6daff89f)

**Comment by PowerShellCrack** [comment]
Yes. We are working very hard at getting the next release of PSD out. Also the new version will include this page. Soon!!!

However currently PSD doesn't support custom pages with dynamic content without modifying the PSDWizard module directly. 

[issue 164](https://github.com/FriendsOfMDT/PSD/issues/164)

**Comment by PowerShellCrack** [comment]
Try the new PSD release 2.30 and let me know if this is still an issue. It is now part of the release.
